app/views/catalog/_show_sidebar.html.erb

```diff
@@ -0,0 +1,16 @@
+<% unless @document[:format][0].empty? %>
```

Comment:
Does every record have a format? If it didn't would this it throw an error? Is every format an array? If it wasn't would it cause an error because [0] doesn't exist? I ask just in case you've already checked.

Reply:
yes, every record should have a format, if none of the rules turns a format then it falls to default "Other". And it should always be an array, Of course both of these are expected unless there is a bug in the logic :)